packages/audit/hooks/audit-route.py

```
@@ -66,8 +68,12 @@ def detect_route(data: dict) -> tuple[str | None, str, str]:
             command = find_first_text(data, {"command", "cmd"})
 
         cmd_lower = command.lower()
+        # 優先順位は audit-cli.py の検出ロジック（codex > antigravity > gemini）に合わせる。
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)